src/comp/ShareButton.tsx

```diff
@@ -71,7 +70,9 @@ const ShareButton = (props: ShareButtonProps) => {
         </Tooltip>
       </div>
       <ShareDialog open={open} handleClose={handleClose}>
-        <OtherShareButton shareButtonType={child} icons={iconsUrl} />
+        {child?.map((icon: IconName, key: number) => (
```

Comment:
```suggestion
        {children?.map((icon: IconName, key: number) => (
```
`child` じゃなくて複数形の `children` にしたい